### PR TITLE
5.x - use nullsafe operator

### PR DESCRIPTION
**Pull request by LordSimal**
Just a small cleanup since we can easily use nullsafe operators for method chaining now

### CODE REVIEW

File: src/TestSuite/IntegrationTestTrait.php [hunk 1]
```diff
@@ -763,7 +763,8 @@ protected function _getBodyAsString(): string
      */
     public function viewVariable(string $name): mixed
     {
-        return $this->_controller ? $this->_controller->viewBuilder()->getVar($name) : null;
+        return $this->_controller?->viewBuilder()
+            ->getVar($name);
```
Comment (othercorey) [inline src/TestSuite/IntegrationTestTrait.php]: I would leave this one line if possible.

File: src/View/Form/EntityContext.php [hunk 2]
```diff
@@ -682,11 +682,9 @@ public function type(string $field): ?string
     {
         $parts = explode('.', $field);
         $table = $this->_getTable($parts);
-        if (!$table) {
-            return null;
-        }
 
-        return $table->getSchema()->baseColumnType(array_pop($parts));
+        return $table?->getSchema()
+            ->baseColumnType(array_pop($parts));
```
Comment (othercorey) [inline src/View/Form/EntityContext.php]: same here

**Comment by othercorey** [comment]
The *only* issue with nullsafe stuff is you can't tell if that path was tested or not since it's all the same line. We just have to rely on static analysis to check types.

**Comment by markstory** [comment]
>  We just have to rely on static analysis to check types.

I think that is a reasonable approach to take. The tools are pretty solid these days and we should be able to rely on them.

**Pull request merged**